src/app/src/theme.js

```diff
@@ -116,6 +116,18 @@ const theme = extendTheme({
             fontFamily: 'heading',
             fontSize: 'xl',
         },
+        loginHeader: {
+            fontFamily: 'heading',
+            fontSize: '3xl',
+            fontWeight: 'bold',
+            alignItems: 'center',
+            bg: 'gray.100',
+        },
+        loginError: {
+            fontSize: 'md',
+            alignItems: 'center',
+            color: 'red',
```

Comment:
This could use some design input from @gatesgodin. It currently isn't legible against the gray background:

![image](https://user-images.githubusercontent.com/1430060/192033388-451e1e54-7668-4572-9735-af332b700cd2.png)

Reply:
Also @gatesgodin if you have an opinion about using placeholder text for email and password instead of labels above the inputs I'm interested in that as well.

Reply:
2399cc093500db6b35606df0627bfe7372873593